docs/sources/setup-grafana/configure-security/configure-authentication/grafana-com/index.md

```diff
@@ -0,0 +1,47 @@
+---
+aliases:
+  - ../../../auth/grafana-com/
+description: Grafana Com Authentication
+title: Configure Grafana Com Authentication
+weight: 500
+---
+
+# Configure Grafana Com authentication
+
+To enable GrafanaCom as your authentication provider. You configure GrafanaCom to generate a client ID and a secret key for you to use.
+
+## Create Grafana Com OAuth keys
+
+To use Grafana Com authentication:
+
+1. Login on to Grafana com
+2. Create an OAuth client by visiting your organization and clicking “OAuth Clients”.
+3. Then click “Add OAuth Client Application”.
+4. Add the URL of your running instance of Grafana.
+5. The next step will give you the client ID and secret.
+
+Below is an example of the configuration:
+
+```ini
+[auth.grafana_com]
+enabled = true
+allow_sign_up = true
+client_id = 450bc21c10dc2194879d
+client_secret = eyJ0Ijoib2F1dGgyYyIhlmlkIjoiNzUwYmMzM2MxMGRjMjE6NDh3OWQiLCJ2IjoiZmI1YzVlYmIwYzFmN2ZhYzZmNjIwOGI1NmVkYTRlNWYxMzgwM2NkMiJ9
+scopes = user:email
+allowed_organizations = sampleorganization
+enabled = true
+
+# prevents the sync of org roles from Grafana.com
+skip_org_role_sync = false
+```
+
+## Skip Org Role Sync
+
+To prevent the sync of org roles from Grafana.com, set `skip_org_role_sync` to `true`. This is useful if you want to use Grafana.com as an identity provider but want to manage the org roles in Grafana.
```

Comment:
I'm a little confused on this one. Why is preventing the sync of org roles a good thing? What happens if they are sync'd? I'd think you'd want them to be sync'd.

Reply:
syncing roles from their idp's is the default for most customers. Some customers ask for this when they have multiple providers and would want to be able to manage roles within Grafana without them being synced from their IDP